client/app.js

```diff
@@ -1,19 +1,18 @@
-/* eslint-env browser */
+const React = require('react');
 
-var React = require('react');
+const { Provider } = require('react-redux');
 
-var store = require('./store')(window.initialStoreData);
+const DOM = require('react-dom');
```

Comment:
I would normally import this as `ReactDOM`, and do it on the line after `React` without a gap.